sale_order_import/wizard/import_sale.py

```diff
@@ -223,167 +488,108 @@ def import_sale_data(self):
                 quotechar='"',
                 delimiter=','
             )
-            fields = next(csv_iterator)
-
-            order_group = fields.index('Group')
-            partner_id = fields.index('Customer')
-            product_id = fields.index('Line Product')
-            line_name = fields.index('Line Description')
-            price_unit = fields.index('Line Unit Price')
-            product_qty = fields.index('Line Qty')
-            taxes_id = fields.index('Line Tax')
-            notes = fields.index('Notes')
-            pricelist_id = fields.index('Pricelist')
-            warehouse_id = fields.index('Warehouse')
+            sheet_fields = next(csv_iterator)
+
+            order_group = sheet_fields.index('Group')
```

Comment:
We should avoid getting internal error when the column is missing in the import file.

Perhaps make the line like `order_group = sheet_fields.index('Group') if 'Group' in sheet_fields else False` or show an error message, and do the same for the other fields as well.